### PR TITLE
TASK: Run acceptance tests also on chrome

### PR DESCRIPTION
**Pull request by markusguenther**

### CODE REVIEW

File: .sauce/config1Dimension.yml [hunk 1]
```diff
@@ -18,15 +18,20 @@ testcafe:
...
-  - name: "One dimensional Tests in Firefox on MacOS"
-    # todo use chrome here and fix ci https://github.com/neos/neos-ui/issues/3591
+  - name: "One dimensional Tests on Firefox on Windows"
     browserName: "firefox"
+    src:
+      - "Tests/IntegrationTests/Fixtures/1Dimension/*.e2e.js"
+    platformName: "Windows 10"
+    screenResolution: "1280x1024"
+  - name: "One dimensional Tests in Edge on MacOS"
+    browserName: "microsoftedge"
```
Comment (mhsdesign) [inline .sauce/config1Dimension.yml:34]: why not chrome? ^^

File: .sauce/config2Dimension.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ kind: testcafe
 sauce:
   region: us-west-1
   concurrency: 1 # Controls how many suites are executed at the same time.
-  retries: 0
+  retries: 3
```
Comment (mhsdesign) [inline .sauce/config2Dimension.yml:6]: retries / smart retries dont work as far as i tried.

File: Tests/IntegrationTests/Fixtures/1Dimension/nodeTreePresets.e2e.js [hunk 3]
```diff
@@ -40,20 +40,6 @@ const SETTINGS_WITH_NODE_TREE_PRESETS = {
                         presets: {
                             'default': {
                                 baseNodeType: 'Neos.Neos:Document,!Neos.TestNodeTypes:Document.Blog,!Neos.TestNodeTypes:Document.BlogArticle'
-                            },
-                            'blog': {
```
Comment (mhsdesign) [inline Tests/IntegrationTests/Fixtures/1Dimension/nodeTreePresets.e2e.js:44]: i think we should try to keep it and make it work on chrome. I dont know why it doesnt work in ci. Locally it works splendidly. We have to debug the http request

File: .sauce/config1Dimension.yml (hunk 1 above)
Comment (markusguenther) [reply .sauce/config1Dimension.yml:34]: It was just a try, in safari we also have issues in the tests ;)

File: Tests/IntegrationTests/Fixtures/1Dimension/nodeTreePresets.e2e.js (hunk 3 above)
Comment (markusguenther) [reply Tests/IntegrationTests/Fixtures/1Dimension/nodeTreePresets.e2e.js:44]: I had the issue also locally, and therefore I changed this and also did not see why we need to change it. But I was not involved in the change back then.

File: .sauce/config2Dimension.yml (hunk 2 above)
Comment (markusguenther) [reply .sauce/config2Dimension.yml:6]: While testing the PR for 8.4 I had the case that one run failed and the retry succeeded. But did not enforce it after that.

**Pull request closed without merging**